### PR TITLE
JML pages additions.

### PR DESCRIPTION
**Pull request by mattmachell**

### CODE REVIEW

File: source/documentation/guides/joiners_movers_leavers.md.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+---
+# <%= current_page.data.title %>
+
+The majority of our Joiners/Movers/Leavers processes can be carried out by using the opg-org-infra Terraform to assign permissions. Some tools like Jira or product-specific admin tools may require additional work. In general the JML process will be a combination of Delivery Manager and WebOps work at the delivery team level.
+
+##Joiners##
+
+New Developer or WebOps team members will need:
+
+- GitHub Access
```
Comment (andrewpearce-digital) [inline source/documentation/guides/joiners_movers_leavers.md.erb:15]: Is this org membership? that's managed by Ops Engineering

File: source/documentation/guides/joiners_movers_leavers.md.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- AWS Access
+- Membership of their delivery teams GitHub team
+- Access to Jira and Confluence
+- Any team specific admin tool access needed in their role.
```
Comment (charlesmmarshall) [inline source/documentation/guides/joiners_movers_leavers.md.erb:19]: Are these outside of the service now ones like o365 / slack etc?

Comment (mattmachell) [reply source/documentation/guides/joiners_movers_leavers.md.erb:19]: Yeah, was thinking things like the LPA Admin Portal (sic)

File: source/documentation/guides/joiners_movers_leavers.md.erb (hunk 1 above)
Comment (mattmachell) [reply source/documentation/guides/joiners_movers_leavers.md.erb:15]: I'll clarify as both org and team membership will be needed and have distinct processes.

**Review by charlesmmarshall — APPROVED** [review]

**Pull request merged**